packages/react/src/themes/teams/components/Tooltip/tooltipContentStyles.ts

```diff
@@ -2,6 +2,21 @@ import { ComponentSlotStylesInput, ICSSInJSStyle } from '../../../types'
 import { TooltipContentProps } from '../../../../components/Tooltip/TooltipContent'
 import { TooltipContentVariables } from './tooltipContentVariables'
 import getPointerStyles from '../../getPointerStyles'
+import svgContent from './tooltipPointerSvgUrl'
```

Comment:
nit: maybe `tooltipPointerSvg` would be more descriptive? I mean, surely, it is more verbose, but it makes the following line much easier to read, as `svgContent` is very abstract

```jsx
backgroundImage: svgContent(v.backgroundColor),
```